src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzerTests.cs

```diff
@@ -3198,6 +3198,232 @@ internal static class UnsupportedWindows6_10_Ios_1_5_11
                     Join(GetFormattedString(MicrosoftNetCoreAnalyzersResources.PlatformCompatibilityFromVersionToVersion, "windows", "10.0", "12.0"), GetFormattedString(MicrosoftNetCoreAnalyzersResources.PlatformCompatibilityFromVersionToVersion, "ios", "9.0", "10.0"))));
         }
 
+        [Fact]
+        public async Task EmptyCallsiteReferencesLocalFunctionNotWarns()
+        {
+            var source = @"
+using System.Runtime.Versioning;
+[SupportedOSPlatform(""Browser"")]
+public class Test
+{
+    [UnsupportedOSPlatform(""browser"")]
```

Comment:
Just curious - does the reverse annotation work? i.e. if I mark the containing assembly or type as unsupported for a platform A and just a single method/property inside it as supported, will we get warnings inside that method/property if it has guarded runtime checks within it?

Reply:
> Just curious - does the reverse annotation work? i.e. if I mark the containing assembly or type as unsupported for a platform A and just a single method/property inside it as supported

Only parent attribute would work, as a child cannot extend parent support supported attribute on M2 will be ignored
```cs
[UnsupportedOSPlatform(""Browser"")] // Deny list => unsupported only on browser but supported on all others 
public class Test
{
    [SupportedOSPlatform(""browser"")] // child cannot extend parent support, annotation willl be ignored
    public void M2 ()  // the above annotation doesn't produce empty set as parent only unsupported on browser but supported on all other 
    {
         // All platforms except browser can access here, so we would warn for platform-specific APIs usage 
    }
}
```
> will we get warnings inside that method/property if it has guarded runtime checks within it?

Good question, this combination is not producing the empty set, so we should warn